packages/react/src/components/TabContent/TabContent.js

```diff
@@ -6,24 +6,46 @@
  */
 
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useCallback, useState } from 'react';
 import classNames from 'classnames';
 import { settings } from 'carbon-components';
+import { selectorTabbable } from '../../internal/keyboard/navigation';
 
 const { prefix } = settings;
 
+/**
+ * Determine if the node within the provided ref contains content that is tabbable.
+ */
+function useTabbableContent() {
+  const [hasTabbableContent, setHasTabbableContent] = useState(false);
+
+  const ref = useCallback((node) => {
+    if (node !== null) {
+      // we don't need to know which element or how many are tabbable, just that there is _something_ tabbable
+      node.querySelector(selectorTabbable)
+        ? setHasTabbableContent(true)
+        : setHasTabbableContent(false);
```

Comment:
```suggestion
      setHasTabbableContent(node.querySelector(selectorTabbable));
```

maybe something like this? seems simpler than a ternary statement